src/App.css

```diff
@@ -1,42 +1,48 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+body {
+  background-color: #264653 !important;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Może przydałby się normalize.css lub reset.css? Poczytaj proszę czym to jest i jak użyć, element starter packu projektowego w sumie to jest.